src/plugins/directory.js

```diff
@@ -0,0 +1,59 @@
+export const classProperties = {
```

Comment:
Most is right. There's also the question of `transform` vs `syntax` (even if we ignore the rarer `preset` and `...-legacy` etc) which is still an editorial decision basically